data/dict/_add.txt

```diff
@@ -5,6 +5,9 @@ Oregon n. Oregon
 Rutgers n. Rutgers
 biting midge n. lúsmý
 Ceratopogonidae n. lúsmý
+kids garage sale n. tombóla
+canteen n. háma
```

Comment:
I think "háma" is only the one in University of Iceland. "Canteen" is already defined:

> canteen n. matsölubúð hermanna; skrín með eldhúss- og borðáhöldum; hermannaflaska

However I don't think this definition is wide enough. Canteen is bacially "mötuneyti", no?

Reply:
and cafeteria